Sources/Domain/SpotMap/SpotMapView.swift

```diff
@@ -60,16 +56,37 @@ struct SpotMapView: View {
         .handle(error: $error)
         .edgesIgnoringSafeArea(.all)
         .task {
-            response = await cache(for: .init(region: region))
             do {
                 let userLocation = try await locationManager.userLocation()
-                region = .init(center: userLocation.coordinate, span: region.span)
-                response = try await query(for: .init(region: region))
+                region = .init(center: userLocation.coordinate, span: mapCoordinateRegion.wrappedValue.span)
+
+                response = await cache(for: .init(region: mapCoordinateRegion.wrappedValue))
+                response = try await query(for: .init(region: mapCoordinateRegion.wrappedValue))
             } catch {
                 self.error = error
             }
         }
     }
+
+    var spots: [SpotsQuery.Data.Spot] {
+        response?.spots ?? []
+    }
+
+    // Workaround of SwiftUI.Map unavoidable warning
+    // SwiftUI.Map(coordinateRegion: $region) is not working well
+    // After update binding property of region on View#task(async) method,
+    // SwiftUI send error about `Modifying state during view update, this will cause undefined behavior`
+    // mapCoordinateRegion avoid this runtime warnings that wrapped and proxy for getting and setting region.
+    // Reference: https://stackoverflow.com/questions/68271517/swiftui-onappear-modifying-state-during-view-update-this-will-cause-undefined-b
+    private var mapCoordinateRegion: Binding<MKCoordinateRegion> {
```

Comment:
SUPER WORKAROUND. View.task 上で `region = .init...` をするだけで `Modifying state during view update, this will cause undefined behavior` が起きる。切り分けると Map(coordinateRegion: Binding) で Bindしている部分が原因だと特定。直接一つ間にBindingを挟むことで警告が消え、挙動も変わってないのでこれでいく